.github/workflows/core_crypto_gpu_benchmark.yml

```diff
@@ -20,7 +20,8 @@ jobs:
   setup-instance:
     name: Setup instance (cuda-core-crypto-benchmarks)
     runs-on: ubuntu-latest
-    if: ${{ (github.event_name == 'push' && github.repository == 'zama-ai/tfhe-rs') || github.event_name == 'workflow_dispatch' }}
+    if: github.event_name != 'schedule' ||
+      (github.event_name == 'schedule' && github.repository == 'zama-ai/tfhe-rs')
```

Comment:
that would trigger bench on push to internal ?

Reply:
There no `push` event on this one. It's a copy/paste error.

Reply:
indeed